help/sources/tutorials/ui/create/analytics/pendo-webhook.md

```diff
@@ -0,0 +1,165 @@
+
+![The streaming endpoint in dataflow activity.](../../../../images/tutorials/create/analytics-pendo-webhook/endpoint-test.png)
+
+Construct a URL using these values following the pattern below:
+```{STREAMING_ENDPOINT}?x-adobe-flow-id={DATAFLOW_ID}```
+
+Example of a constructed webhook URL: ```https://dcs.adobedc.net/collection/0c61859cc71939a0caf01123f91b2fc52589018800ad46b6c76c2dff3595ee95```
+
+## Next steps {#next-steps}
+
+By following this tutorial you have successfully [configured a dataflow to bring data into Platform](https:/experienceleague.adobe.com/docs/experience-platform/sources/ui-tutorials/dataflow/crm.html).
+
+Next, login to your account on [[!DNL Pendo]](https:/pendo.io/) and create a webhook by following the [[!DNL Pendo] create a webhook](https://support.pendo.io/hc/en-us/articles/360032285012-Webhooks#create-a-webhook-0-4) tutorial. You need to create a URL Webhook.
+
+Navigate to the **[!UICONTROL Settings]** > **[!UICONTROL Integrations]** > **[!UICONTROL Webhook]** page and use the [URL](#get-streaming-endpoint-url) that was constructed, in the **[!UICONTROL URL]** field.
+![[DNL Pendo] UI screenshot showing the webhook endpoint field](../../../../images/tutorials/create/analytics-pendo-webhook/webhook.png)
+
+Refer to the [!DNL Pendo] [documentation](https://support.pendo.io/hc/en-us/articles/360032285012-Webhooks#create-a-webhook-0-4) to select individual event categories to choose which event categories you want to send into Platform.
+
+## Additional resources
+
+The sections below provide additional resources that you can refer to when using the [DNL Pendo] source.
+
+### Validation {#validation}
+
+To validate that you have correctly set up the source and [!DNL Pendo] messages are being ingested, follow the steps below:
+
+* You can check the [!DNL Pendo] **[!UICONTROL Reports]** > **[!UICONTROL Chat History]** page to identify the events being captured by [!DNL Pendo].
+![Pendo UI screenshot showing chat history](../../../../images/tutorials/create/analytics-pendo-webhook/pendo-events.png)
+
+* In the Platform UI, select **[!UICONTROL View Dataflows]** beside the [!DNL Pendo] card menu on the sources catalog. Next, select **[!UICONTROL Preview dataset]** to verify the data that was ingested for the webhooks that you have configured within [!DNL Pendo].
+![Platform UI screenshot showing ingested events](../../../../images/tutorials/create/analytics-pendo-webhook/platform-dataset.png)
+
+### Errors and troubleshooting {#errors-and-troubleshooting}
+
+* When checking a dataflow run, you might encounter the following error message: `The message can't be validated ... uniqueID:expected minLength:1, actual 0].`
+    ![Platform UI screenshot showing error.](../../../../images/tutorials/create/analytics-pendo-webhook/error.png)
+
+    * To fix this error, verify that the *uniqueID* mapping has been set up. Refer to the [Mapping](#mapping) section for guidance.
+
+### Documentation
```

Comment:
I think we can remove this H3 and incorporate the help center link within Errors and troubleshooting. Maybe just a final sentence like: "For more information, visit the [[!DNL Pendo] Help Center](https://www.pendo.io/help-center/)